.circleci/config.yml

```diff
@@ -445,9 +447,12 @@ jobs:
       name: win/vs2019
       shell: bash.exe -eo pipefail
     steps:
+      # while these two steps are also done in build-windows, they are needed
```

Comment:
Do you know why?

Reply:
IIUC the build phases "persist" the emscripten and emsdk artifacts but they don't persist any system packages.. so each job needs to install any system packages it needs (I think).

In theory we shouldn't need python once emsdk has been installed since the emsdk version of python should be used over the system one.  But that would be a separate change i think.

Reply:
No idea, but it errors on not finding python, without installing it "twice"...

Reply:
I don't really see why you added this comment the system packages were needed in both places regardless of this change.. but not harm I guess.

Reply:
Well, the comment would have saved me a bunch of time... but yeah, maybe that's just reflecting my lack of understanding of how circleCI persists stuff. I can remove if you want?